Sources/Fuzzilli/Minimization/BlockReducer.swift

```diff
@@ -155,8 +158,7 @@ struct BlockReducer: Reducer {
                 break
             }
         }
-
-        if candidates.count == 4 && verifier.tryNopping(candidates, in: &code) {
+        if (candidates.count - tryCatch.numBlocks) == 0 && verifier.tryNopping(candidates, in: &code) {
```

Comment:
I think this would need to be 1 instead of 0, but let's maybe just simplify this to `if removedLastTryBlockInstruction && verifier.tryNopping(candidates, in: &code) { ...`, then add `var removedLastTryBlockInstruction = false` above the preceding loop, and set it to true when appending the index. That should make it more obvious what's going on